apps/www/components/Audio/helpers/getImageCropURL.ts

```diff
@@ -1,9 +1,6 @@
-export type AudioCoverPropsOptions = {
-  x: number
-  y: number
-  width: number
-  height: number
-}
+import { Crop } from '#graphql/republik-api/__generated__/gql/graphql'
```

Comment:
Just wondering if we should move all the types to AudioPlayer/types instead of importing some from the generated gql files and some from types